iac-extensions/arm/src/main/java/org/sonar/iac/arm/parser/bicep/BicepLexicalGrammar.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * SonarQube IaC Plugin
+ * Copyright (C) 2021-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.iac.arm.parser.bicep;
+
+import com.sonar.sslr.api.GenericTokenType;
+import java.util.Arrays;
+import org.sonar.iac.common.parser.grammar.LexicalConstant;
+import org.sonar.iac.common.parser.grammar.Punctuator;
+import org.sonar.sslr.grammar.GrammarRuleKey;
+import org.sonar.sslr.grammar.LexerlessGrammarBuilder;
+
+public enum BicepLexicalGrammar implements GrammarRuleKey {
+
+  /**
+   * Lexical
+   */
+  EOF,
+
+  /**
+   * SPACING
+   */
+  SPACING,
+  EOL,
+
+  FILE,
+  STATEMENT,
+  TARGET_SCOPE_DECLARATION,
+  EXPRESSION,
```

Comment:
```suggestion
  
  /**
   * Statements
   */
  STATEMENT,
  TARGET_SCOPE_DECLARATION,
  
  /**
   * Expressions
   */
  EXPRESSION,
```
Maybe this can help later on to differentiate better between statements and expressions